spec/views/hq/partners/_form.html.erb_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/partners/_form.html.erb", type: :view do
+
+  let(:provinces) { Province.all }
+  let(:statuses) { Status.all }
```

Comment:
Is there a performance hit from using `.all` rather than taking just one or two? I suspect not.